app/src/main/java/com/rob729/quiethours/util/StoreSession.kt

```diff
@@ -61,4 +61,13 @@ object StoreSession {
     fun readLong(key: String): Long {
         return sharedPreferences.getLong(key, 0)
     }
+
```

Comment:
There is already a function for writeBoolean and readBoolean then why add the below functions?

Reply:
Ok.. i'll fix it